### ISSUE TITLE
Full Screen showing toolbar and other layout

### ISSUE THREAD

**Issue by Naguchennai** [kakarrot/ExoPlayerView#1]
If set video height 200dp full screen showing all other layouts. also tootbar.  also if set vide height matchparent this also showing toolbar when fullscreen

**Comment by kakarrot** [comment]
Yes, you are right, this verison you should hide or show toolbar and other views by yourself.
Maybe you should override this method in Activity, like:
@Override  
    public void onConfigurationChanged(Configuration newConfig) {  
        super.onConfigurationChanged(newConfig);  

```
    if(newConfig.orientation==Configuration.ORIENTATION_PORTRAIT){  
        //show your views;
    }  
    if(newConfig.orientation==Configuration.ORIENTATION_LANDSCAPE){  
        //hide your views.
    }  
}
```

of course, I will continue to optimize this project，thanks for your issue.
